Makefile

```diff
@@ -65,7 +65,7 @@ endif
 ## Ensures NPM dependencies are installed without having to run this all the time.
 webapp/.npminstall:
 ifneq ($(HAS_WEBAPP),)
-	cd webapp && $(NPM) install
+	cd webapp && $(NPM) install --verbose
```

Comment:
We've updated CI so this is no longer necessary. Can we remove this change here?

@hanzei There may be other lingering open PRs that have this change